src/models/euDccCertificates/index.ts

```diff
@@ -74,7 +74,7 @@ const genEuDccCertificates = async (
       expiryDays: euSigner.expiryDays,
       patientDetails: {
         name: parsedFhirBundle.patient.fullName,
-        dateOfBirth: isoToDateOnlyString(parsedFhirBundle.patient.birthDate),
+        dateOfBirth: parsedFhirBundle.patient.birthDate.split("T")[0], // validated birthDate format can be only YYYY-MM-DD or YYYY-MM or YYYY or ISO-8601 format.
```

Comment:
I foresee one possible issue if a provider uses ISO datetime string that incorporates timing information for DOB. Then because of the timezone, the DOB might be +/- 1 day difference.

**For example**:
My birthdate in Singapore is `2022-05-13`. But in ISO-8601, it is written as `2022-05-12T16:00:00.000Z`. So just removing the T will result in: `2022-05-12` (which isn't correct)

![image](https://user-images.githubusercontent.com/37650399/168230720-fe08e7f2-45e2-43c9-98f7-460a342cad53.png)